storage.tf

```diff
@@ -0,0 +1,41 @@
+# Storage bucket for public files.
+
+resource "google_storage_bucket" "public" {
+  name                        = "${var.subdomain_prefix}storage.${var.top_domain}"
+  location                    = var.storage_bucket_location
+  storage_class               = var.storage_bucket_class
+  uniform_bucket_level_access = false
+
+  force_destroy = true
+
+  cors {
+    origin          = ["*"]
+    method          = ["*"]
+    response_header = ["*"]
+    max_age_seconds = 3600
+  }
+}
+
+resource "google_storage_bucket_access_control" "public" {
+  bucket = google_storage_bucket.public.id
```

Comment:
The public bucket lists the files if you link to the root or to a directory. It would be nicer if only the files would be accessible and not directory listings. Is there a way to do this somehow?